articles/quickstart/spa/angular2/_includes/_centralized_login.md

```diff
@@ -86,7 +86,6 @@ export class AuthService {
   public handleAuthentication(): void {
     this.auth0.parseHash((err, authResult) => {
       if (authResult && authResult.accessToken && authResult.idToken) {
-        window.location.hash = '';
```

Comment:
This line triggered a navigation event that canceled the `router.navigate(['/home']); ` call on line 90/91 made.